### ISSUE TITLE
Doc changes for 0.13.0

### ISSUE THREAD

**Issue by micgibso** [eclipse/codewind#2946]
**Codewind version:** 0.13.0
**OS:**

**Description of the enhancement:**
This is a summary record of all documentation changes in the Codewind 0.13.0 release:
[Brief description] [Link to issue] [Author]

**Comment by j-c-berger** [comment]
Please assign me, thanks!

**Comment by MelHopper** [comment]
To assign me.

**Comment by micgibso** [comment]
Michael
1.  Fix remote, https://github.com/eclipse/codewind/issues/2926, Michael
2.

**Comment by sishida** [comment]
Sarah's latest doc changes:

**Comment by MelHopper** [comment]
Mel's doc changes for 0.13.0:

1. Eclipse Che section: improve clarity, flow, signposting. Particular work done on adding browser certificates and image registries topics. Delivered under:
- https://github.com/eclipse/codewind-docs/pull/573

**Comment by j-c-berger** [comment]
Jake's doc changes for 0.13.0

- [#613-CRW quick guide](https://github.com/eclipse/codewind-docs/pull/613): in progress. PR is up and waiting on dev review before making the new commit and getting SVT to review. 

- [ #487-Minikube quick guide](https://github.com/eclipse/codewind-docs/pull/487): on hold. Bug with Minikube so Minikube must be updated before we progress with documentation. 

-  #2360 : in progress. Will make PR once I have screenshots and learn how to insert screenshots. 

-  #2711 : in progress. 

-  #2973  #2993: in progress. PRs with corrections on both guides will be up shortly.
